### PR TITLE
(MODULES-1638) Remove prompt parameter

### PR DESCRIPTION
**Pull request by elyscape**
[JIRA ticket here](https://tickets.puppetlabs.com/browse/MODULES-1638).  This also bumps the version to 0.2.0-SNAPSHOT as it is a breaking change.

### CODE REVIEW

**Comment by joshcooper** [comment]
:+1:

/cc @cyberious

**Comment by cyberious** [comment]
Will test soon. But otherwise looks good to me.

**Comment by elyscape** [comment]
I went ahead and rebased just in case that was helpful.

**Comment by elyscape** [comment]
Ping.  Any update on this?

**Comment by elyscape** [comment]
Rebased onto master.

**Comment by elyscape** [comment]
Rebased again to take #66 into account.  Pinging @joshcooper, @cyberious, @ferventcoder, and @hunner for review.

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-reboot",
-  "version": "0.1.10-SNAPSHOT",
+  "version": "0.2.0-SNAPSHOT",
```
Comment (ferventcoder) [inline metadata.json]: Not sure what this change is here for

File: spec/unit/provider/reboot/windows_spec.rb [hunk 2]
```diff
@@ -99,17 +99,6 @@
       provider.reboot
     end
 
-    it "does not include the interactive flag by default" do
-      provider.expects(:async_shutdown).with(Not(includes('/i')))
-      provider.reboot
-    end
```
Comment (ferventcoder) [inline spec/unit/provider/reboot/windows_spec.rb]: Let's not remove this. We need to verify it is never added.

**Comment by ferventcoder** [comment]
Mostly +1, just a couple of thoughts.

File: metadata.json (hunk 1 above)
Comment (elyscape) [inline metadata.json]: This is a breaking change so I figured we should bump the version number.  Honestly, I think once we merge this and #59, we should bump the version to 1.0.0.

File: spec/unit/provider/reboot/windows_spec.rb (hunk 2 above)
Comment (elyscape) [inline spec/unit/provider/reboot/windows_spec.rb]: Added back in.

**Comment by elyscape** [comment]
Pinging @joshcooper, @cyberious, @ferventcoder.  Can we get some movement on this?

**Comment by ferventcoder** [comment]
Looks like this needs rebased and I think @cyberious is going to merge it with fire

**Comment by cyberious** [comment]
@elyscape please rebase and update Readme after

**Comment by elyscape** [comment]
@cyberious Done.

**Pull request merged**